src/centralserver/admin-ui/src/views/Members/Member/Subsystems/MemberSubsystems.vue

```diff
@@ -0,0 +1,153 @@
+              <v-icon class="sorting">icon-Sorting-arrow</v-icon>
+            </th>
+            <th>{{ $t('members.member.subsystems.servercode') }}</th>
+            <th>{{ $t('members.member.subsystems.serverOwner') }}</th>
+            <th>{{ $t('members.member.subsystems.status') }}</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Management</td>
+            <td class="xrd-clickable">NIIS-SS1</td>
+            <td></td>
+            <td class="status">
+              <v-icon :small="true" :color="colors.Success100"
+                >icon-Checked</v-icon
+              >
+              Registered
+            </td>
+            <td class="subsystem-actions">
+              <span class="xrd-clickable">Delete</span>
+            </td>
+          </tr>
+          <tr class="unregistered-subsystem">
+            <td>RR</td>
+            <td></td>
+            <td></td>
+            <td class="status"><v-icon>icon-Error</v-icon> Unregistered</td>
+            <td class="subsystem-actions">
+              <span class="xrd-clickable">Register</span>
+              <span class="xrd-clickable">Delete</span>
+            </td>
+          </tr>
+          <tr>
+            <td rowspan="2">DHX</td>
+            <td class="xrd-clickable">NIIS-SS1</td>
+            <td class="xrd-clickable">NIIS</td>
+            <td class="status">
+              <v-icon :small="true" :color="colors.Success100"
+                >icon-Checked</v-icon
+              >
+              Registered
+            </td>
+            <td class="subsystem-actions">
+              <span class="xrd-clickable">Delete</span>
+            </td>
+          </tr>
+          <tr>
+            <td class="xrd-clickable">NIIS-FUTU</td>
+            <td class="xrd-clickable">FUTU</td>
+            <td class="status">
+              <v-icon :small="true" :color="colors.Success100"
+                >icon-In-progress</v-icon
+              >
+              Pending approval
+            </td>
+            <td class="subsystem-actions">
+              <span class="xrd-clickable">Approve</span>
+              <span class="xrd-clickable">Delete</span>
+            </td>
+          </tr>
+        </tbody>
+      </xrd-table>
+    </v-card-text>
+  </v-card>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { Colors } from '@/global';
+
+/**
+ * Component for Member Subsystem
+ */
+export default Vue.extend({
+  name: 'MemberSubsystems',
+  data() {
+    return {
+      colors: Colors,
+    };
+  },
+});
+</script>
+
+<style scoped lang="scss">
+@import '../../../../assets/colors';
+
+.card-corner-button {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.status {
+  text-transform: uppercase;
+  font-weight: bold;
+}
+
+.subsystem-actions {
+  text-align: right;
+
+  .xrd-clickable {
+    margin-left: 10px;
+  }
+}
+
+.sorting {
+  margin-bottom: 5px;
+}
```

Comment:
I don't necessarily find it very convenient that a css class is called `sorting` and it only adds 5px bottom margin. Vuetify's `mb-1` utility class already adds 4px bottom margin so that could possibly be used instead.

At some point it would probably be good to come up with some sort of guide about the styles. For example there are a lot of duplicate css classes (with minor differencies) in the security server UI also: search for example `.clickable-link` from the codebase.

Reply:
Agreed. Don't know why I've left it that way as I've used those vuetifys helper css classes.

Reply:
Fixed. I guess I used 'sorting' as it gives a better hint on what is that class used for (positioning sorting arrow) but I removed it as unnecessary.